src/functions.php

```diff
@@ -110,6 +110,22 @@ function is_first_key_operator($document)
     return (isset($firstKey[0]) && $firstKey[0] === '$');
 }
 
+/**
+ * Returns whether we are currently in a transaction
+ *
+ * @internal
+ * @param array  $options  Command options
+ * @return bool
```

Comment:
I actually think that might have been the only place `bool` was used instead of `boolean`. The pedant in me would see this changed for consistency (noting that we might go in an entirely different direction after implementing code standard checks down the line).

Reply:
That is correct. I've changed it to `boolean` for the time being. The coding standard mentioned above includes a sniff (https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Sniffs/TypeHints/LongTypeHintsSniff.php) that forces short type names to be used (`int` and `bool` vs. `integer` and `boolean`, respectively). I suppose we'll discuss that when the time comes.